avtozip/store/api/resources.py

```diff
@@ -0,0 +1,16 @@
+from tastypie.resources import ModelResource
+from store.models import Product
+from django.http import Http404
+
+
+class ProductResource(ModelResource):
+    class Meta:
+        queryset = Product.objects.all()
+        allowed_methods = ['post', 'put', 'get', 'patch', 'delete']
+        filtering = {
+            'name': ('exact', 'icontains',),
+        }
+
+    def get_schema(self, request, **kwargs):
+        # Forbidden schema request
+        raise Http404('Forbidden');
```

Comment:
HTTP404 error means something not found, but in the description you provided that this information is `forbidden`. Please omit the error message by now completely